MetaMask/metamask-extension
# Fix misaligned icons in 'Connected sites' modal

Fixes #19901: Connected sites modal misaligned icons

Replaced `<SiteIcon>` with `<AvatarFavicon>` in `<ConnectedSitesList>`

### Before

![19901 before](https://github.com/MetaMask/metamask-extension/assets/4392888/7885e77c-8954-4824-8331-22306d7ec35c)

### After

![border-muted](https://github.com/MetaMask/metamask-extension/assets/4392888/124452e6-319f-4360-93b9-0095004656f1)

## Manual Testing Steps

- Log in to MetaMask.
- Connect the wallet to at least one site (if not already).
- Open the 'Connected sites' modal.
- Observe all listed site icons are centered within their circular container element.

## Pre-merge author checklist


## Pre-merge reviewer checklist
